README.md

```diff
@@ -6,37 +6,118 @@
 
-The current implementation allows to preview .dcm files and display their corresponding metadata in a sidebar on request. It offers image manipulation operations such as zoom, rotate, flip, invert and reset of the image preview. The UI is implemented in responsive manner and adapts the size of the image preview and the way how metadata is displayed to the screen size of the device.
 
-## Quick reference
+## Table of Contents
 
-- **Where to file issues:**\
-  [owncloud/web-app-dicom-viewer](https://github.com/owncloud/web-app-dicom-viewer/issues)
+* [Functionalities of DICOM Viewer Web Extension](#functionalities-of-dicom-viewer-web-extension)
+* [Adding DICOM Viewer to Your oCIS Installation](#adding-dicom-viewer-to-your-ocis-installation)
+  * [Adding DICOM Viewer to the oCIS Deployment Example](#adding-dicom-viewer-to-the-ocis-deployment-example)
+  * [Manual App Installation](#manual-app-installation)
+* [Build and Run DICOM Viewer for Development](#build-and-run-dicom-viewer-for-development)
+* [Contributing to DICOM Viewer Web Extension](#contributing-to-dicom-viewer-web-extension)
+* [Copyright](#copyright)
 
-- **Supported architectures:**\
-  `amd64`
 
-## Installation
+## Functionalities of DICOM Viewer Web Extension
 
-### 1. Install dependencies
+The current release allows to preview .dcm files within oCIS and display their corresponding metadata in a sidebar on request. It offers image manipulation operations such as zoom in and out, rotation, flipping, colour inversion and reset on the image preview. The app UI is implemented in a responsive manner and adapts the size of the image preview and the way how metadata is displayed to the screen size of the device.
 
-```
+<img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/1.png" alt="app functionalities" style="width:48%; height:auto;"> &nbsp; &nbsp; <img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/4.png" alt="display of metadata" style="width:48%; height:auto;">
+
+_The extension allows to preview a DICOM image and it's most important metadata. On request, all corresponding metadata of the file are displayed in the sidebar._
+
+<img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/2.png" alt="app functionalities" style="width:48%; height:auto;"> &nbsp; &nbsp; <img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/3.png" alt="display of metadata" style="width:48%; height:auto;">
+
+_The extension allows to zoom, rotate and flip the preview of the image. Inverting the colors of the preview is also supported._
```

Comment:
```suggestion
_The extension allows zooming, rotating and flipping the preview of the image. Inverting the colors of the preview is also supported._
```

If you write "The extension allows the user to zoom, rotate..." then that works in English. But if "the user" is omitted we need to say "zooming, rotating..."

Reply:
will add "the user". I think these are some interactions the user does more actively/consciously, where as previewing is just what is expected :)